src/minimap.asm

```diff
@@ -0,0 +1,214 @@
+;=======================================================
+;Patches to support the minimap
+;=======================================================
+
+org $809B51
+    JMP $9BFB    ; skip drawing auto reserve icon and normal energy numbers and tanks during HUD routine
+
+org $82AED9      ; routine to draw auto reserve icon on HUD from equip screen
+    NOP : NOP : NOP
+
+org $82AEAF      ; routine to remove auto reserve icon on HUD from equip screen
+    NOP : NOP : NOP
+
+org $809AF3
+    JSL mm_initialize_minimap
+
+org $90A91B
+    LDA !ram_minimap : BNE .update_minimap
```

Comment:
Tracking CPU impact of the minimap logic (when minimap is off) that occurs every frame.  There's some impact to other areas such as ih_update_hud_code but that only runs at specific times.

This LDA : BNE adds 6 CPU cycles when minimap is off